{
	"id": "RLSA-2021:1762",
	"advisories": [
		{
			"content": {
				"id": "RLSA-2021:1762",
				"title": "Moderate: virt:rhel and virt-devel:rhel security, bug fix, and enhancement update",
				"description": "Kernel-based Virtual Machine (KVM) offers a full virtualization solution for Linux on numerous hardware platforms. The virt:Rocky Linux module contains packages which provide user-space components used to run virtual machines using KVM. The packages also provide APIs for managing and interacting with the virtualized systems.\n\nSecurity Fix(es):\n\n* libvirt: double free in qemuAgentGetInterfaces() in qemu_agent.c (CVE-2020-25637)\n\n* QEMU: heap buffer overflow in msix_table_mmio_write() in hw/pci/msix.c (CVE-2020-27821)\n\n* QEMU: ide: atapi: OOB access while processing read commands (CVE-2020-29443)\n\n* QEMU: heap buffer overflow in iscsi_aio_ioctl_cb() in block/iscsi.c may lead to information disclosure (CVE-2020-11947)\n\n* QEMU: reachable assertion failure in net_tx_pkt_add_raw_fragment() in hw/net/net_tx_pkt.c (CVE-2020-16092)\n\n* QEMU: infinite loop in e1000e_write_packet_to_guest() in hw/net/e1000e_core.c (CVE-2020-25707)\n\n* QEMU: assertion failure through usb_packet_unmap() in hw/usb/hcd-ehci.c (CVE-2020-25723)\n\n* QEMU: e1000e: infinite loop scenario in case of null packet descriptor (CVE-2020-28916)\n\n* QEMU: slirp: out-of-bounds access while processing ARP/NCSI packets (CVE-2020-29129, CVE-2020-29130)\n\nFor more details about the security issue(s), including the impact, a CVSS score, acknowledgments, and other related information, refer to the CVE page(s) listed in the References section.\n\nAdditional Changes:\n\nFor detailed information on changes in this release, see the Rocky Linux 8.4 Release Notes linked from the References section.",
				"severity": [
					{
						"type": "vendor",
						"source": "errata.rockylinux.org",
						"vendor": "MODERATE"
					}
				],
				"references": [
					{
						"source": "MITRE",
						"url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-11947"
					},
					{
						"source": "MITRE",
						"url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-16092"
					},
					{
						"source": "MITRE",
						"url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-25637"
					},
					{
						"source": "MITRE",
						"url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-25707"
					},
					{
						"source": "MITRE",
						"url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-25723"
					},
					{
						"source": "MITRE",
						"url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-27821"
					},
					{
						"source": "MITRE",
						"url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-28916"
					},
					{
						"source": "MITRE",
						"url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-29129"
					},
					{
						"source": "MITRE",
						"url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-29130"
					},
					{
						"source": "MITRE",
						"url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-29443"
					},
					{
						"source": "Red Hat",
						"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1384241"
					},
					{
						"source": "Red Hat",
						"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1798463"
					},
					{
						"source": "Red Hat",
						"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1828952"
					},
					{
						"source": "Red Hat",
						"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1834281"
					},
					{
						"source": "Red Hat",
						"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1837495"
					},
					{
						"source": "Red Hat",
						"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1843852"
					},
					{
						"source": "Red Hat",
						"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1846975"
					},
					{
						"source": "Red Hat",
						"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1850680"
					},
					{
						"source": "Red Hat",
						"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1859494"
					},
					{
						"source": "Red Hat",
						"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1860283"
					},
					{
						"source": "Red Hat",
						"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1872854"
					},
					{
						"source": "Red Hat",
						"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1874304"
					},
					{
						"source": "Red Hat",
						"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1874780"
					},
					{
						"source": "Red Hat",
						"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1876297"
					},
					{
						"source": "Red Hat",
						"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1876742"
					},
					{
						"source": "Red Hat",
						"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1880418"
					},
					{
						"source": "Red Hat",
						"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1880546"
					},
					{
						"source": "Red Hat",
						"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1881037"
					},
					{
						"source": "Red Hat",
						"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1884531"
					},
					{
						"source": "Red Hat",
						"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1893895"
					},
					{
						"source": "Red Hat",
						"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1898579"
					},
					{
						"source": "Red Hat",
						"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1901837"
					},
					{
						"source": "Red Hat",
						"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1902231"
					},
					{
						"source": "Red Hat",
						"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1902651"
					},
					{
						"source": "Red Hat",
						"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1902960"
					},
					{
						"source": "Red Hat",
						"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1903064"
					},
					{
						"source": "Red Hat",
						"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1910220"
					},
					{
						"source": "Red Hat",
						"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1910267"
					},
					{
						"source": "Red Hat",
						"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1910326"
					},
					{
						"source": "Red Hat",
						"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1912765"
					},
					{
						"source": "Red Hat",
						"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1917446"
					},
					{
						"source": "Red Hat",
						"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1918708"
					},
					{
						"source": "errata.rockylinux.org",
						"url": "https://errata.rockylinux.org/RLSA-2021:1762"
					}
				],
				"published": "2021-05-18T06:02:26Z"
			}
		}
	],
	"vulnerabilities": [
		{
			"content": {
				"id": "CVE-2020-11947",
				"severity": [
					{
						"type": "cvss_v31",
						"source": "MITRE",
						"cvss_v31": {
							"vector": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:C/C:L/I:N/A:N",
							"base_score": 3.8,
							"base_severity": "LOW",
							"temporal_score": 3.8,
							"temporal_severity": "LOW",
							"environmental_score": 3.8,
							"environmental_severity": "LOW"
						}
					}
				],
				"cwe": [
					{
						"source": "MITRE",
						"cwe": [
							"CWE-131->CWE-122"
						]
					}
				],
				"references": [
					{
						"source": "MITRE",
						"url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-11947"
					}
				]
			}
		},
		{
			"content": {
				"id": "CVE-2020-16092",
				"severity": [
					{
						"type": "cvss_v31",
						"source": "MITRE",
						"cvss_v31": {
							"vector": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:C/C:N/I:N/A:L",
							"base_score": 3.8,
							"base_severity": "LOW",
							"temporal_score": 3.8,
							"temporal_severity": "LOW",
							"environmental_score": 3.8,
							"environmental_severity": "LOW"
						}
					}
				],
				"cwe": [
					{
						"source": "MITRE",
						"cwe": [
							"CWE-617"
						]
					}
				],
				"references": [
					{
						"source": "MITRE",
						"url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-16092"
					}
				]
			}
		},
		{
			"content": {
				"id": "CVE-2020-25637",
				"severity": [
					{
						"type": "cvss_v31",
						"source": "MITRE",
						"cvss_v31": {
							"vector": "CVSS:3.1/AV:L/AC:H/PR:H/UI:N/S:U/C:H/I:H/A:H",
							"base_score": 6.4,
							"base_severity": "MEDIUM",
							"temporal_score": 6.4,
							"temporal_severity": "MEDIUM",
							"environmental_score": 6.4,
							"environmental_severity": "MEDIUM"
						}
					}
				],
				"cwe": [
					{
						"source": "MITRE",
						"cwe": [
							"CWE-415"
						]
					}
				],
				"references": [
					{
						"source": "MITRE",
						"url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-25637"
					}
				]
			}
		},
		{
			"content": {
				"id": "CVE-2020-25707",
				"severity": [
					{
						"type": "cvss_v31",
						"source": "MITRE",
						"cvss_v31": {
							"vector": "CVSS:3.1/AV:L/AC:H/PR:H/UI:N/S:C/C:N/I:N/A:L",
							"base_score": 2.5,
							"base_severity": "LOW",
							"temporal_score": 2.5,
							"temporal_severity": "LOW",
							"environmental_score": 2.5,
							"environmental_severity": "LOW"
						}
					}
				],
				"cwe": [
					{
						"source": "MITRE",
						"cwe": [
							"CWE-835"
						]
					}
				],
				"references": [
					{
						"source": "MITRE",
						"url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-25707"
					}
				]
			}
		},
		{
			"content": {
				"id": "CVE-2020-25723",
				"severity": [
					{
						"type": "cvss_v31",
						"source": "MITRE",
						"cvss_v31": {
							"vector": "CVSS:3.1/AV:L/AC:L/PR:H/UI:N/S:C/C:N/I:N/A:L",
							"base_score": 3.2,
							"base_severity": "LOW",
							"temporal_score": 3.2,
							"temporal_severity": "LOW",
							"environmental_score": 3.2,
							"environmental_severity": "LOW"
						}
					}
				],
				"cwe": [
					{
						"source": "MITRE",
						"cwe": [
							"CWE-617"
						]
					}
				],
				"references": [
					{
						"source": "MITRE",
						"url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-25723"
					}
				]
			}
		},
		{
			"content": {
				"id": "CVE-2020-27821",
				"severity": [
					{
						"type": "cvss_v31",
						"source": "MITRE",
						"cvss_v31": {
							"vector": "CVSS:3.1/AV:L/AC:L/PR:H/UI:N/S:C/C:L/I:L/A:L",
							"base_score": 5.7,
							"base_severity": "MEDIUM",
							"temporal_score": 5.7,
							"temporal_severity": "MEDIUM",
							"environmental_score": 5.7,
							"environmental_severity": "MEDIUM"
						}
					}
				],
				"cwe": [
					{
						"source": "MITRE",
						"cwe": [
							"CWE-787->CWE-122"
						]
					}
				],
				"references": [
					{
						"source": "MITRE",
						"url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-27821"
					}
				]
			}
		},
		{
			"content": {
				"id": "CVE-2020-28916",
				"severity": [
					{
						"type": "cvss_v31",
						"source": "MITRE",
						"cvss_v31": {
							"vector": "CVSS:3.1/AV:L/AC:H/PR:H/UI:N/S:C/C:N/I:N/A:L",
							"base_score": 2.5,
							"base_severity": "LOW",
							"temporal_score": 2.5,
							"temporal_severity": "LOW",
							"environmental_score": 2.5,
							"environmental_severity": "LOW"
						}
					}
				],
				"cwe": [
					{
						"source": "MITRE",
						"cwe": [
							"CWE-835"
						]
					}
				],
				"references": [
					{
						"source": "MITRE",
						"url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-28916"
					}
				]
			}
		},
		{
			"content": {
				"id": "CVE-2020-29129",
				"severity": [
					{
						"type": "cvss_v31",
						"source": "MITRE",
						"cvss_v31": {
							"vector": "CVSS:3.1/AV:L/AC:H/PR:H/UI:N/S:C/C:L/I:N/A:N",
							"base_score": 2.5,
							"base_severity": "LOW",
							"temporal_score": 2.5,
							"temporal_severity": "LOW",
							"environmental_score": 2.5,
							"environmental_severity": "LOW"
						}
					}
				],
				"cwe": [
					{
						"source": "MITRE",
						"cwe": [
							"CWE-125"
						]
					}
				],
				"references": [
					{
						"source": "MITRE",
						"url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-29129"
					}
				]
			}
		},
		{
			"content": {
				"id": "CVE-2020-29130",
				"severity": [
					{
						"type": "cvss_v31",
						"source": "MITRE",
						"cvss_v31": {
							"vector": "CVSS:3.1/AV:L/AC:H/PR:H/UI:N/S:C/C:L/I:N/A:N",
							"base_score": 2.5,
							"base_severity": "LOW",
							"temporal_score": 2.5,
							"temporal_severity": "LOW",
							"environmental_score": 2.5,
							"environmental_severity": "LOW"
						}
					}
				],
				"cwe": [
					{
						"source": "MITRE",
						"cwe": [
							"CWE-125"
						]
					}
				],
				"references": [
					{
						"source": "MITRE",
						"url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-29130"
					}
				]
			}
		},
		{
			"content": {
				"id": "CVE-2020-29443",
				"severity": [
					{
						"type": "cvss_v31",
						"source": "MITRE",
						"cvss_v31": {
							"vector": "CVSS:3.1/AV:L/AC:H/PR:H/UI:N/S:C/C:L/I:N/A:L",
							"base_score": 3.9,
							"base_severity": "LOW",
							"temporal_score": 3.9,
							"temporal_severity": "LOW",
							"environmental_score": 3.9,
							"environmental_severity": "LOW"
						}
					}
				],
				"cwe": [
					{
						"source": "MITRE",
						"cwe": [
							"CWE-125"
						]
					}
				],
				"references": [
					{
						"source": "MITRE",
						"url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-29443"
					}
				]
			}
		}
	],
	"data_source": {
		"id": "rocky-errata",
		"raws": [
			"vuls-data-raw-rocky-errata/2021/RLSA-2021:1762.json"
		]
	}
}
